lib/irb/command/copy.rb

```diff
@@ -0,0 +1,68 @@
+# frozen_string_literal: true
+
+require 'mkmf'
+
+module IRB
+  module Command
+    class Copy < Base
+      category "TODO"
+      description "TODO"
+
+      help_message(<<~HELP)
+        Usage: copy (input)
+      HELP
+
+      def execute(arg)
+        output = irb_context.workspace.binding.eval(arg)
```

Comment:
I think we need to apply the [output inspection method](https://github.com/ruby/irb/blob/master/lib/irb/context.rb#L676-L678) here so the copied output will be closer to what the users may expect. 

For example, the `output` here by default will be the `to_s` result of the evaluation result, but IRB's default output inspector [pretty prints](https://github.com/ruby/irb/blob/master/lib/irb/inspector.rb#L116-L118) the result inspect.

Additionally, we need to make sure colorizing is NOT applied to the copied value.

Reply:
The problem with using the pretty printer is that colorizing output is [hardcoded into it](https://github.com/Shopify/irb/blob/sin.add-copy-command/lib/irb/color_printer.rb#L37-L53), and I don't think [we want it in our clipboard](https://screenshot.click/16-29-kmq50-u4g6w.jpg).

Unless there's a pretty printer that I've missed, the options are:

1. Grab colorized output, strip color before copying. Least invasive, and will respect the currently configured inspector.
- Install a new pretty printer analogous to [the default color printer](https://github.com/Shopify/irb/blob/sin.add-copy-command/lib/irb/color_printer.rb) that does basically the same thing but doesn't colorize output. Don't like this one because we're conflating two concepts and hardcoding a command to use a specific kind of inspection method.
- Like above, `color_printer.rb` to accept a parameter that turns colorizing on / off. Most invasive.

I like 1, but will defer.